src/Assertions.jl

```diff
@@ -282,7 +282,7 @@ verbosity scopes by calling the function [`add_verbosity_scope`](@ref).
 julia> add_verbosity_scope(:MyScope1);
 
 julia> get_verbosity_level(:MyScope1)
-0
+4
```

Comment:
This should stay `0`, but you ran the tests too often :)

Reply:
Yes indeed, I am currently fixing it, I just noticed that it was wrong when pushing the new commits.